scripts/influxd_wrapper.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+/usr/bin/influxd
```

Comment:
Should this be:
```suggestion
/usr/bin/influxd &
```
?

Reply:
And if so, do we need to track the pid so we can somehow tell systemd about it? Or is it tracked automatically since it's a child of the wrapper script?

Reply:
Probably